### PR TITLE
Changes stormtrooper ammo to 10g slugs

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Syndicate stormtroopers now rapid fire 10g slugs instead of buckshot. This deals the same damage, but makes getting hit less laggy.
/:cl:

Right now, syndie stormtroopers rapid fire buckshot shells with zero spread. Getting hit with a salvo of 18 pellets will spam your screen and lag your client to hell (up to 36 messages at once).

I changed them to use 10g slugs instead - this will still kill you dead in record time, but in a less spammy/laggy fashion.
Unless armor calculations change it, the damage should be the same (six 12.5 brute buckshot pellets vs. a single 72.5 brute slug).

**tl;dr if you get hit by a full salvo while wearing armor, your client will only have to process 6 instead of 36 messages. Damage is the same as before.**

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
nerf

**Comment by 81Denton** [comment]
>nerf
I should have checked the values, looks like it's 72.5 brute -> 40 burn. Uhh let me find something that does similar damage

**Comment by juanototo2** [comment]
Just make them shoot a penetrator bullet that does that amount of damage and dismembers :^)

**Comment by SpaceManiac** [comment]
> lag your client
Yep, like 4 seconds of freeze if you're hit directly, worse if you're wearing armor since that doubles the number of messages.

**Comment by 81Denton** [comment]
I added 10g slugs that deal the same damage as the magic buckshot SMG. Unless armor calculations somehow change it.

**Comment by 81Denton** [comment]
@kevinz000 
wat

**Comment by Robustin** [comment]
Honestly its still probably a nerf because its much easier to dodge a single projectile than a wave of death even if some of the pellets miss you it'll still 2-shot you easy. The caravan staying ultra-lethal is the priority, I don't want to go back to the days of it people coming back with a bag of guns every round.

**Comment by 81Denton** [comment]
@Robustin 
It's not a wave of death, stormtroopers fire regular projectiles that impact as 8 buckshot pellets each. Hence the lag and message spam.

Stormtrooper:
syndie01

Regular syndie:
syndie02

**Comment by ExcessiveUseOfCobblestone** [comment]
Why are you making a new subtype instead of overwriting lagshot?

**Comment by 81Denton** [comment]
Because my brain is smooth and I honestly have no idea how to.

**Comment by ExcessiveUseOfCobblestone** [comment]
you don't edit damage in casing, that's in bullet.

**Comment by ExcessiveUseOfCobblestone** [comment]
@kevinz000 why aren't these spreading anyways? they have variance.

**Comment by vuonojenmustaturska** [comment]
works for me
